CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing Guidelines
+
+Deviation from this guide is off limits to those who do not wish to die a most painful death.
+
+## Code Style
+
+- Two space indent
+- 88-character line limit
+
```

Comment:
A few things I'd add:
- Spaces instead of tabs
- Line continuations should be at least +4
- Don't use /* ... */ style comments except for Javadocs